oak_dice/src/evidence.rs

```diff
@@ -32,6 +32,13 @@ pub const PUBLIC_KEY_SIZE: usize = 256;
 /// The maximum size of a serialized CWT certificate.
 pub const CERTIFICATE_SIZE: usize = 1024;
 
+/// The name of the kernel command-line parameter that is used to send the physical address of the
+/// Stage 0 DICE data struct.
+pub const DICE_DATA_CMDLINE_PARAM: &str = "oak-dice";
+
+/// The magic number used to identify the Stage 0 DICE data in memory.
+pub const STAGE0_MAGIC: u64 = u64::from_le_bytes(*b"oak.dice");
```

Comment:
How nice that "Oak Dice" is exactly 8 bytes :)

Reply:
Yes, indeed.